nifi-registry/nifi-registry-extensions/nifi-registry-aws/pom.xml

```diff
@@ -30,7 +30,7 @@
     </modules>
 
     <properties>
-        <aws-java-sdk-version>2.5.9</aws-java-sdk-version>
+        <aws-java-sdk-version>2.17.106</aws-java-sdk-version>
```

Comment:
If this needs to match another version value but the two can't use the same maven property (because they're not sharing a parent), is it worth adding a comment here and in the other `pom.xml` so there's more chance of people realising that updating one means they need to update the other?

Reply:
Since everything is in the same repository now, the root pom is the common parent so we can have this property defined there and shared if we want. The reason I didn't do this initially was because I view this fix as somewhat temporary, the real solution is to figure out a way for all of these AWS/Azure/GCP JARs to not end up in the lib directory of NiFi Registry (or NiFi bootstrap). We didn't intend for NiFi Registry's S3BundlePersistenceProvider to need to use the same AWS client version as NiFi's sensitive property provider, but currently it has to due to the way things are setup.